MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+# Asset Matrix
+
+Maximo HP Utilities includes matrix view for determing the distribution of assets based on scores and then take action to avoid imminent failures.
+
+In this exercise the Reliability Engineer uses HP Utilities to:
+	
+1. [View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health](#view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health)
```

Comment:
@caolan-ibm I didn't see an anchor tag with #view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health) in this matrix.md    Where will this link take them other than the current page?  Is that  your intention?  See create project for an example in WatsonStudio.md  ## Create a Project in Watson Studio
<a name="create-project"></a>

Reply:
Hi @carlosibm the link is working as the #view-matrix-for-criticality-and-xxx is a syntax supported by markdown, there is no need to add `<a name="get-credentials"></a>`. I already tested on my local by ./build_all_mkdocs.sh

Reply:
It is linked to the Section in the same page